packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -30,10 +30,18 @@ private const val SOURCE_HEIGHT_KEY = "height"
 private const val SOURCE_SCALE_KEY = "scale"
 
 @SuppressLint("ViewConstructor")
-class ExpoImageView(context: ReactContext, private val requestManager: RequestManager, private val progressInterceptor: OkHttpClientProgressInterceptor) : AppCompatImageView(context) {
+class ExpoImageView(
+  context: ReactContext,
+  private val requestManager: RequestManager,
+  private val progressInterceptor: OkHttpClientProgressInterceptor
+) : AppCompatImageView(context) {
   private val eventEmitter = context.getJSModule(RCTEventEmitter::class.java)
   private val outlineProvider = OutlineProvider(context)
-  private var borderDrawable: Lazy<BorderDrawable> = lazy {
+
+  private var propsChanged = false
+  private var loadedSource: GlideUrl? = null
+
+  private val borderDrawable = lazy {
```

Comment:
Note that I replaced `var` with `val` here to prevent accidental overrides of the `Lazy` wrapper.